stations/map.js

```diff
@@ -23,6 +23,7 @@ const selectPinlet = station => `${station.status}-${station.speed}`;
  */
 
 export const loadStation = stations => {
+  document.getElementById('stationAmount').innerHTML = 'Showing ' + stations.length + ' stations';
```

Comment:
You can also keep the text and change dynamically only the number ;) this way less content strings in the code.